### DISCUSSION TITLE
refresh form data at  a set of interval

### DISCUSSION THREAD

**Discussion by liza-nt** [filamentphp/filament#8649] [Help]
we have poll() for table content that refreshs at some interval of time can we do that same form data

**Comment by leandrocfe** [comment]
You could use `wire:poll` but why and where are you trying to do this? Could you provide more info please?

**Comment by liza-nt** [comment]
I have this form i want to use poll here in form
```
  public static function form(Form $form): Form
    {
        return $form
            ->schema([
                Section::make('Match Details')
                    ->schema([
                        TextInput::make('en_match_id')
                            ->label(strval(__('khelnow-sports.matches.form.input.en_match_id'))),
                        TextInput::make('status')->label(strval(__('khelnow-sports.matches.form.input.status'))),
                        Select::make('competition_id')
                        ....

**Reply by leandrocfe** [reply]
You can use `extraAttributes`
`Section::make('Match Details')->extraAttributes(['wire:poll' => 'updateFields'])`

```php
//CreatePage.php, EditPage.php ...
public function updateFields(): void
{
    //do something..
}
```

**Reply by liza-nt** [reply]
can you tell me what to do in this function if i want to refresh my content

**Reply by leandrocfe** [reply]
I think you can use `$this->refreshFormData(['your_field'])`
If you want to replace `$this->data['your_field'] = 'new value'`

**Reply by DeviJv** [reply]
@leandrocfe how to set sec interval poll?

**Reply by leandrocfe** [reply]
`wire:poll.15s` //15 seconds

**Reply by DeviJv** [reply]
@leandrocfe Thx Bro

**Comment by liza-nt** [comment]
but how can i use $this->refreshFormData(['your_field']) in view page i can not get field there right??

**Reply by leandrocfe** [reply]
`$this->refreshFormData(['field_1', 'field_2']);`
Didn't it work?

**Reply by liza-nt** [reply]
no it did not work

**Reply by leandrocfe** [reply]
Share the whole code
